common/chatdata.go

```diff
@@ -89,38 +91,51 @@ type DataMessage struct {
 	Type    MessageType
 }
 
```

Comment:
All the changes to this file should be in a separate pull request.

Reply:
Allmost

Reply:
Done, the Other PR is on it's way.

Reply:
https://github.com/zorchenhimer/MovieNight/pull/143